packaging/flavorgen/cmd/root.go

```diff
@@ -52,13 +80,50 @@ func RunRoot(command *cobra.Command) error {
 	if err != nil {
 		return errors.Wrapf(err, "error accessing flag %s for command %s", flavorFlag, command.Name())
 	}
+	outputDir, err := command.Flags().GetString(outputDirFlag)
+	if err != nil {
+		return errors.Wrapf(err, "error accessing flag %s for command %s", outputDirFlag, command.Name())
+	}
+	var flavors []string
+	if flavor != "" {
+		flavors = append(flavors, flavor)
+	} else {
+		flavors = allFlavors
+	}
+	generateMultiFlavors := len(flavors) > 1
+	for _, f := range flavors {
+		objs, err := generateSingle(f)
+		if err != nil {
+			return err
+		}
+
+		yamlFileName := flavorMappings[f] + ".yaml"
+		if outputDir == "" {
```

Comment:
I'm thinking about if we should use `-` for printing to stdout. And "" instead just writes to the current directory. WDYT? Let's reflect the behaviour at least also at the flag description.

Reply:
That's a good idea which makes the behavior more predictable.